packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.2.11
+
+* Adds echo boolean integration test.
```

Comment:
Test-only changes don't need version updates or changelog entries, the tooling just still needs to be taught about pigeon test structure. For now we can use the override labels.